src/api/services/configs.ts

```diff
@@ -335,8 +335,8 @@ export const getConfigInsights = async <T>(
     }`;
   }
   return resolve(
-    ConfigDB.get<T>(
-      `/config_analysis?select=*,config:configs(id,name,config_class,type)&config_id=eq.${configId}${paginationQueryParams}`,
+    ConfigDB.get<ConfigTypeInsights[]>(
+      `/config_analysis?select=id,source,analyzer,analysis_type,message,severity,status,analysis,first_observed,config:configs(id,name,config_class,type)&config_id=eq.${configId}${paginationQueryParams}`,
```

Comment:
We shouldn't be including analysis as its pretty large - It should only show on the popup window